### DISCUSSION TITLE
Is there any way to enable autocomplete for options of the command in terminal?

### DISCUSSION THREAD

**Discussion by 677230756E64** [projectdiscovery/subfinder#914] [Q&A]
I initially had a PATH issue and my terminal didn't recognize the command 'subfinder', but I solved it. But the options for the command aren't recognized, meaning if there's some long-worded option and it's unique I can't just start typing it in, hit Tab and make it autocomplete. Is there any way around this?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
